lib/dashboard-widgets/home_screen.dart

```diff
@@ -1,15 +1,14 @@
-// ignore_for_file: library_private_types_in_public_api
-
 import 'dart:async';
+
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 import 'package:visitplusapp/dashboard-widgets/CategoryIcon.dart';
 import 'package:visitplusapp/dashboard-widgets/doctorCard.dart';
 
 import 'package:visitplusapp/dashboard-widgets/bottom_navigation_bar.dart';
-import 'package:visitplusapp/authentication-widgets/signin_screen.dart';
 import 'package:visitplusapp/doctor-widgets/doctor_profile_screen.dart';
+import 'package:visitplusapp/map%20widgets/map_screen.dart';
```

Comment:
@Priyankabaride Have you used any space in this directory name "map%20widgets"?

Reply:
yes i have map screen class in that which is used to add data to database